pkg/serverless/proc/proc.go

```diff
@@ -196,3 +198,84 @@ func getNetworkData(path string) (*NetworkData, error) {
 	}
 
 }
+
+type FileDescriptorMaxData struct {
+	MaximumFileHandles float64
+}
+
+// GetFileDescriptorMaxData returns the maximum limit of file descriptors the function can use
+func GetFileDescriptorMaxData() (*FileDescriptorMaxData, error) {
+	return getFileDescriptorMaxData()
+}
+
+func getFileDescriptorMaxData() (*FileDescriptorMaxData, error) {
```

Comment:
We should have some unit tests for this function and the file descriptor use data function below. You can use [the network metric unit tests](https://github.com/DataDog/datadog-agent/blob/a8df5c5eab15bcb2a649c3475dda9fcba34d53bf/pkg/serverless/proc/proc_test.go#L119) as reference